freeipa/src/test/java/com/sequenceiq/freeipa/converter/stack/CreateFreeIpaRequestToStackConverterTest.java

```diff
@@ -119,34 +119,22 @@ void testConvertForInstanceGroupsWhendiskEncryptionSetIdIsPresent() {
                 .build());
         when(crnService.createCrn(ACCOUNT_ID, CrnResourceDescriptor.FREEIPA)).thenReturn("resourceCrn");
         when(stackAuthenticationConverter.convert(source.getAuthentication())).thenReturn(new StackAuthentication());
-        //when(cachedEnvironmentClientService.getByCrn(source.getEnvironmentCrn())).thenReturn(environmentResponse);
-        when(instanceGroupConverter.convert(
-                any(InstanceGroupRequest.class),
-                any(NetworkRequest.class),
-                anyString(),
-                any(Stack.class),
-                any(FreeIpaServerRequest.class),
-                any(DetailedEnvironmentResponse.class),
-                anyString()))
-                .thenReturn(new InstanceGroup());
+        when(instanceGroupConverter.convert(any(),
+                any(),
+                any(),
+                any(),
+                any(),
+                any(),
+                mapCaptorForEncryption.capture())).thenReturn(new InstanceGroup());
         when(networkConverter.convert(any(NetworkRequest.class))).thenReturn(new Network());
         when(telemetryConverter.convert(source.getTelemetry())).thenReturn(new Telemetry());
         when(backupConverter.convert(source.getTelemetry())).thenReturn(new Backup());
         when(entitlementService.internalTenant(ACCOUNT_ID)).thenReturn(Boolean.FALSE);
         when(costTagging.prepareDefaultTags(any())).thenReturn(new HashMap<>());
-        ReflectionTestUtils.setField(underTest, "userGetTimeout", 10L);
-        ReflectionTestUtils.setField(underTest, "defaultGatewayCidr", Set.of());
         Future<String> owner = CompletableFuture.completedFuture("dummyUser");
 
         underTest.convert(source, environmentResponse, ACCOUNT_ID, owner, "crn1", CloudPlatform.AZURE.name());
-        verify(instanceGroupConverter).convert(
-                any(InstanceGroupRequest.class),
-                any(NetworkRequest.class),
-                anyString(),
-                any(Stack.class),
-                any(FreeIpaServerRequest.class),
-                any(DetailedEnvironmentResponse.class),
-                anyString());
+        assertEquals(mapCaptorForEncryption.getValue().get(DISK_ENCRYPTION_SET_ID), "dummyDiskEncryptionSetId");
```

Comment:
This test case is failing. I am not sure what broke. I see that this assertion was removed during multi-AZ implementation. https://github.com/hortonworks/cloudbreak/pull/11215#discussion_r699708326

CC: @doktoric @juhi-09 @lajosrodek

Reply:
Strangely removing this mocking fixes the issue, `when(networkConverter.convert(any(NetworkRequest.class))).thenReturn(new Network());`